package.json

```diff
@@ -2,22 +2,24 @@
   "name": "eslint-find-rules",
   "version": "0.0.0-semantically-released",
   "description": "Find built-in ESLint rules you don't have in your custom config.",
-  "main": "src/lib/rule-finder.js",
+  "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
+    "lint": "xo",
     "test": "mocha --recursive",
-    "lint": "eslint --ignore-pattern test/fixtures .",
     "update-contributors": "all-contributors generate",
     "commit": "git-cz",
     "validate": "npm-run-all --parallel lint cover --sequential check-coverage",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
+    "build": "babel src -d dist --presets es2015",
```

Comment:
i'd recommend adding `rimraf` and `mkdirp` and doing `rimraf dist && mkdirp dist && babel …` here, otherwise if a `src` file is deleted, the corresponding `dist` file could end up remaining in the package.